### ISSUE TITLE
get_historical_klines does not work

### ISSUE THREAD

**Issue by LeoIannacone** [sammchardy/python-binance#173]
This code:
```
client.get_historical_klines(symbol='TRXETH', interval=Client.KLINE_INTERVAL_1MONTH, start_str='1 year ago')
```
returns to me:

```
.................lib/python2.7/site-packages/binance/client.pyc in get_historical_klines(self, symbol, interval, start_str, end_str)
    667 
    668                 # update our start timestamp using the last value in the array and add the interval timeframe
--> 669                 start_ts = temp_data[len(temp_data) - 1][0] + timeframe
    670             else:
    671                 # it wasn't listed yet, increment our start date

TypeError: unsupported operand type(s) for +: 'int' and 'NoneType'
```

**Comment by jcageman** [comment]
That is a bug in helpers.py where the method interval_to_milliseconds is not correctly implemented for the month interval. Any other kline interval would work.

**Comment by sammchardy** [comment]
Hi @LeoIannacone this is a bug which I've fixed in v0.6.5

**Comment by LeoIannacone** [comment]
Thanks!
